test/images/scripts/virt-builder-fedora

```diff
@@ -32,6 +32,8 @@ if [ "$#" -gt 3 ]; then
     subscription_args="--mkdir /root/.rhel \
       --upload $subscription_path/login:/root/.rhel/login \
       --upload $subscription_path/pass:/root/.rhel/pass"
+else
+    subscription_args="--touch /root/.skip_repos"
```

Comment:
Interesting that it also took you a while to see, @mvollmer - I had the same issue

I recommend leaving the mechanisms exactly as proposed here, since we have to add a file locally on the vm either way. If we use as proposed, @jscotka won't have to change anything.

Maybe we can just add a comment right above this if-else? For example:
```
# If we have subscription details as an arg, upload those into the vm
# Otherwise ensure subscription is skipped during setup
```